ui/src/app/utils/access-utils.tsx

```diff
@@ -73,3 +78,7 @@ export const useIsUserDisabled = () => {
   }, [authLoaded, isSignedIn]);
   return disabled;
 };
+
+export const getAccessModules = (profile: Profile, moduleName: AccessModule): AccessModuleStatus => {
```

Comment:
the name doesn't reflect what it returns

Reply:
`getAccessModuleStatusByName`?